src/main/java/org/openstreetmap/atlas/geography/atlas/multi/MultiAtlasBorderFixer.java

```diff
@@ -264,7 +266,16 @@ protected void fixBorderIssues()
 
                 // Persist relations
                 relationsToUpdate.addAll(candidateRelations);
-                relationMembersToUpdate.putAll(candidateRelationMembers);
```

Comment:
NOTE: This is not the only place where `putAll` is called. There may be set wipeout happening in other spots.